Adding instructions on how to Create an Asset-Only Gem (#782)
o3de/o3de.org: content/docs/user-guide/gems/development/creating/_index.md

o3de create-gem -gp <path to create gem at>
```

This will create your Gem with a standard file structure and CMake files created from [templates](https://github.com/o3de/o3de/tree/development/Templates).

> 

Reviewer: Suggestion - Navigate users to the CLI Reference page so they can quickly find out how to run the `o3de` command for their platform. This is helpful for users who aren't familiar with the `o3de` tool. The linked _CLI Reference_ page is currently a placeholder page, but PR https://github.com/o3de/o3de.org/pull/763 will replace that.

```suggestion

For more information on how to use the `o3de` tool, refer to [Project Configuration CLI Reference](/docs/user-guide/project-config/cli-reference/)

```

Author: That makes sense, if we can point to another documentation page it keeps the user in the same frame of mind at looking at the o3de.org documentation pages.

  This will be the active target during the invocation of your Gem's `CMakeLists.txt`.
* You can use the functions available in the core O3DE build system. See the contents of the `cmake` directory in source.
* Avoid the use of `file(DOWNLOAD ...)`. The package system of O3DE is a robust replacement, and should be used instead.

# Creating an Asset-Only Gem

> When a gem is created using the o3de `o3de create-gem -gp <path to create gem at>` command, by default it creates a Gem that can used for packaging code and assets.
> This Gem is actually created using the "DefaultGem" template.
> There is a way to create a gem that is only meant for providing asset only and not code.
> This can can be done by specifying the "AssetGem" template through the `create-gem` command
> ```cmd
> o3de create-gem -gp <path to create gem at> -tn AssetGem
> # or
> o3de create-gem -gp <path to create gem at> -tp <engine-root>\Templates\AssetGem
> ```
> 

Reviewer: - Reworded your paragraph to use an active voice, instead of a passive voice. This complies with the O3DE Style Guide. ("When you create a Gem" (active) vs. "When a gem is created" (passive))
- Other minor suggestions improve reading flow. 
- Gem is a proper nound and must be capitalized. 

```suggestion
When you create a Gem without a specified template by using the command `o3de create-gem -gp <path to create a gem at>`, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:

```cmd
o3de create-gem --gem-path <path to create gem at> --template-name AssetGem
# or
o3de create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```

```

Author: Thanks for the edit, I'll commit that suggestion.

## Gem Templates

> The list of Gem Templates that are provided with O3DE can be found in the [Templates](https://github.com/o3de/o3de/tree/development/Templates) directory in the main O3DE repo

Reviewer: - Switched to active voice. 
```suggestion
O3DE provides a list of Gem templates, which can be found in the [Templates](https://github.com/o3de/o3de/tree/development/Templates) directory in the O3DE repository.
```

  This will be the active target during the invocation of your Gem's `CMakeLists.txt`.
* You can use the functions available in the core O3DE build system. See the contents of the `cmake` directory in source.
* Avoid the use of `file(DOWNLOAD ...)`. The package system of O3DE is a robust replacement, and should be used instead.

# Creating an Asset-Only Gem

When you create a Gem without a specified template by using the command `o3de create-gem -gp <path to create a gem at>`, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:

```cmd
o3de create-gem --gem-path <path to create gem at> --template-name AssetGem
# or

> o3de create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem

Reviewer 2: Complete the code block with three backquotes.

Author: Fixed

> ## Gem Templates

Reviewer 2: ```suggestion

## Gem Templates

```
